tools-local/tasks/BuildFPMToolPreReqs.Netstandard2_0.cs

```diff
@@ -0,0 +1,581 @@
+        private static readonly string Package_Name_String = nameof(ConfigJson.Package_Name).ToLower();
+        private static readonly string Install_Root_String = nameof(ConfigJson.Install_Root).ToLower();
+        private static readonly string Install_Doc_String = nameof(ConfigJson.Install_Doc).ToLower();
+        private static readonly string Short_Description_String = nameof(ConfigJson.Short_Description).ToLower();
+        private static readonly string Long_Description_String = nameof(ConfigJson.Long_Description).ToLower();
+        private static readonly string Homepage_String = nameof(ConfigJson.Homepage).ToLower();
+        private static readonly string Release_String = nameof(ConfigJson.Release).ToLower();
+        private static readonly string Control_String = nameof(ConfigJson.Control).ToLower();
+        private static readonly string CopyRight_String = nameof(ConfigJson.CopyRight).ToLower();
+        private static readonly string License_String = nameof(ConfigJson.License).ToLower();
+        private static readonly string Rpm_Dependencies_String = nameof(ConfigJson.Rpm_Dependencies).ToLower();
+        private static readonly string Directories_String = nameof(ConfigJson.Directories).ToLower();
+        private static readonly string Install_Man_String = nameof(ConfigJson.Install_Man).ToLower();
+        private static readonly string After_Install_Source_String = nameof(ConfigJson.After_Install_Source).ToLower();
+        private static readonly string After_Remove_Source_String = nameof(ConfigJson.After_Remove_Source).ToLower();
+        private static readonly string Package_Conflicts_String = nameof(ConfigJson.Package_Conflicts).ToLower();
+
+        private static readonly string Package_Version_String = nameof(Release.Package_Version).ToLower();
+        private static readonly string Package_Revision_String = nameof(Release.Package_Revision).ToLower();
+        private static readonly string Urgency_String = nameof(Release.Urgency).ToLower();
+        private static readonly string Changelog_Message_String = nameof(Release.Changelog_Message).ToLower();
+
+        private static readonly string Priority_String = nameof(Control.Priority).ToLower();
+        private static readonly string Section_String = nameof(Control.Section).ToLower();
+        private static readonly string Architecture_String = nameof(Control.Architecture).ToLower();
+
+        private static readonly string Type_String = nameof(License.Type).ToLower();
+        private static readonly string Full_Text_String = nameof(License.Full_Text).ToLower();
+
+        private static readonly string RpmDependency_Package_Name_String = nameof(RpmDependency.Package_Name).ToLower();
+        private static readonly string RpmDependency_Package_Version_String = nameof(RpmDependency.Package_Version).ToLower();
+    }
+
+    /// <summary>
+    /// Model classes for reading and storing the JSON. 
+    /// </summary>
+    public class ConfigJson
```

Comment:
As with the other PR; there's a lot of duplication between this file and the alternate for this file.